about.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js" defer></script>
+</head>
+<body>
+    <section class="header-section">
+        <div class="main-containers">
+            <nav class="inner-header">
+                <li class="inner-list"><img src="images/facebook.png" alt=""></li>
+                <li class="inner-list"><img src="images/twitter.png" alt=""></li>
+                <li class="inner-list"><a href="#">English</a></li>
+                <li class="inner-list"><a href="#">My page</a></li>
+                <li class="inner-list"><a href="#">Logout</a></li>
+            </nav>
+        </div>
+            <header>
+                <div class="main-containers">
+                    <div class="header-container">
+                      <nav class="header">
+                        <div class="my-logo">
+                            <a href="#"><img class="logo" src="images/Ifeanyi_Logo_1B-01-removebg-preview.png" alt=""></a>
+                        </div>
+                        <div class="hambuger">
+                          <div class="bar"></div>
+                          <div class="bar"></div>
+                          <div class="bar"></div>
+                        </div>
+                        <ul class="nav-list">
+                          <li class="nav-item about"><a href="#">About</a></li>
+                          <li class="nav-item"><a href="#section1">Program</a></li>
+                          <li class="nav-item"><a href="#section2">join</a></li>
+                          <li class="nav-item"><a href="#section3">Sponsor</a></li>
+                          <li class="nav-item"><a href="#">News</a></li>
+                          <li class="nav-item"><a class="campaign" href="#">Campaign</a></li>
+                        </ul>
+                      </nav>
+                    </div>
+                </div>
+            </header>
+    </section>
+    <main>
+        <section class="about-section">
+           <div class="main-container">
+               <div class="about-title">
+                  <h1 class="ifeanyi">Ifeanyi305 Virtual Learning in Schools: Pros and Cons. is a platform 
+                    you can access anytime,anywhere.
+                  </h1>
+                  <div class="description2">
+                    By far the most popular 
+                    approach today is online learning. According
+                     to the Sloan Consortium, online enrollments continue to 
+                     grow at rates faster than for the broader student 
+                     population and institutes of higher education expect the 
+                     rate of growth to continue increasing.
+                  </div>
```

Comment:
- [ ] Kindly use the correct semantic tag which is `p` tag as this is a paragraph